<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hall</class>
 <widget class="QWidget" name="Hall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Go-Online</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/Go.ico</normaloff>
    <normalon>../Images/Go.ico</normalon>
    <disabledoff>../Images/Go.ico</disabledoff>
    <disabledon>../Images/Go.ico</disabledon>
    <activeoff>../Images/Go.ico</activeoff>
    <activeon>../Images/Go.ico</activeon>
    <selectedoff>../Images/Go.ico</selectedoff>
    <selectedon>../Images/Go.ico</selectedon>../Images/Go.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 3, 3);</string>
  </property>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Send">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please enter your message.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>520</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Matching">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string>Match</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
